azure/pull-request.pipeline.yml

```diff
@@ -32,3 +32,15 @@ steps:
 - script: |
     CI=true yarn test-cypress-with-test-api
   displayName: 'Yarn run cypress (e2e) tests'
+
+- task: PublishTestResults@2
```

Comment:
Should we add this also to the master-to-test.yml pipeline?

Reply:
Yes,  once I get it to work 👍